infra-gen2/infra-common/src/auth-user-extensions/custom-sender-lambda.ts

```diff
@@ -44,9 +44,6 @@ export function addCustomSenderLambda({
         "custom-email-sender.js"
       ),
       runtime: Runtime.NODEJS_18_X,
-      bundling: {
-        nodeModules: ["@aws-crypto/client-node"],
-      },
```

Comment:
Note: This was causing an issue deploying the infra, and is not needed. It am not sure why it was not causing an issue previously.

Reply:
thanks for the clean up.